discovery-provider/alembic/versions/a6d2e50a8efa_remove_old_trending.py

```diff
@@ -0,0 +1,30 @@
+"""remove_old_trending
+
+Revision ID: a6d2e50a8efa
+Revises: 7104383ac0fe
+Create Date: 2022-01-21 04:00:29.311676
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "a6d2e50a8efa"
+down_revision = "7104383ac0fe"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+    query = """
+        begin;
+            DROP MATERIALIZED VIEW IF EXISTS trending_params_aSPET;
+        commit;
+    """
+    conn.execute(query)
+
+
+def downgrade():
+    # No down migration as this is destructive
```

Comment:
Do we eventually want to be able to downgrade completely? Seems like hard to execute in reality but I know @cheran-senthil is pretty adamant about it. 

What if there's something more destructive in the new one?

Reply:
Not sure what you mean -- I don't think there is a reasonable downgrade here unless the upgrade persists the table we would need to recover?

Reply:
Like -- this migration is an (intentional) data loss one, so you can't go back.